services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/extensions/dynamodb/mappingclient/DatabaseOperation.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.extensions.dynamodb.mappingclient;
+
+import java.util.function.Function;
+
+import software.amazon.awssdk.annotations.SdkPublicApi;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+
+/**
+ * Interface for a single operation that can be executed against a mapped database. These operations do not operate
+ * on a specific table or index, and may reference multiple tables and indexes (eg: batch operations). Conceptually an
+ * operation maps 1:1 with an actual DynamoDb call.
+ *
```

Comment:
Missing `<p>` tag. Seems it's missing on other places as well. Can we fix it?(or add a backlog item for it)

Reply:
As discussed punting for now.